test/unit/c11_test_shmem_atomic_add.c

```diff
@@ -41,6 +41,14 @@
 enum op { ADD = 0, ATOMIC_ADD, CTX_ATOMIC_ADD, FADD, ATOMIC_FETCH_ADD,
           CTX_ATOMIC_FETCH_ADD };
 
+#ifdef ENABLE_DEPRECATED_TESTS
```

Comment:
Might want to file an issue to hook this in so we don't forget?  (or not!  :hear_no_evil:)

Reply:
Done: #542